static/src/js/components/Layout/Layout.jsx

```diff
@@ -236,7 +236,7 @@ const Layout = ({ className, displayNav }) => {
                             naked
                             Icon={FaUserAlt}
                           >
-                            {`${user.name} `}
+                            {`${user?.name} `}
```

Comment:
Were we seeing errors related to this?

Reply:
URS had some issues yesterday and it would not return the user name (MMT crashed in sit, uat, and prod).